taxi/views.py

```diff
@@ -52,3 +53,43 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class CarsCreateView(LoginRequiredMixin, generic.edit.CreateView):
```

Comment:
No need to import this from `edit`
Simplify to just `generic.CreateView` and in all places below